### PR TITLE
fixing JENKINS-47631: Plugin cannot access SV project on Jenkins slave

### PR DESCRIPTION
**Pull request by pavelchuchma**
- refactoring SV build steps to be able to run on Jenkins slave nodes via MasterToSlaveCallable

Please Make sure these boxes are checked before submitting your pull request - Thanks ahead!

- [x ] Proper pull request title - concise and clear for others.
- [x] Proper pull request short description - clear and concise (as it should appear in the Jira ticket) for other developers and users.
- [x] Proper Jira ticket - Number, Link in pull request description. - **JENKINS-47631**
- [x] The PR can is merged on your machine without any conflicts.
- [x] The PR can is built on your machine without any (new) warnings.
- [x] The PR passed sanity tests by you / QA / DevTest / Good Samaritan.
- [ ] Add unit tests with new features.
- [ ] If you added any dependency to the POM - Please update grount

### CODE REVIEW

File: src/main/java/com/microfocus/application/automation/tools/run/AbstractRemoteRunner.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package com.microfocus.application.automation.tools.run;
```
Comment (grount) [inline src/main/java/com/microfocus/application/automation/tools/run/AbstractRemoteRunner.java]: License missing and please move this class to SV folder

Comment (pavelchuchma) [reply src/main/java/com/microfocus/application/automation/tools/run/AbstractRemoteRunner.java]: Missing license is not nice. I'm going to fix it. Anyway, are you able to configure automatic checks for it?

Comment (grount) [reply src/main/java/com/microfocus/application/automation/tools/run/AbstractRemoteRunner.java]: Do you mean adding the license checks as part of the CI?

File: src/main/java/com/microfocus/application/automation/tools/run/AbstractSvRemoteRunner.java [hunk 2]
```diff
@@ -21,13 +41,13 @@
 import hudson.model.TaskListener;
 import jenkins.security.MasterToSlaveCallable;
 
-public abstract class AbstractRemoteRunner<T extends AbstractSvRunModel> extends MasterToSlaveCallable<String, Exception> {
+public abstract class AbstractSvRemoteRunner<T extends AbstractSvRunModel> extends MasterToSlaveCallable<String, Exception> {
```
Comment (grount) [inline src/main/java/com/microfocus/application/automation/tools/run/AbstractSvRemoteRunner.java]: Besides the name, I also require that each new class will be in its folder.
For your case it should be under: hpe-application-automation-tools-plugin/src/main/java/com/microfocus/application/automation/tools/sv/
After /sv/ you can define your own package hirearchy 
For example see [Octane's](https://github.com/jenkinsci/hpe-application-automation-tools-plugin/tree/latest/src/main/java/com/microfocus/application/automation/tools/octane) hirearchy.

**Pull request merged**